leaf/extensions/tags.py

```diff
@@ -402,6 +402,55 @@ async def transfer_tag(
                     )
                 )
 
+    @app_commands.describe(tag="The tag you wish to claim.")
+    @app_commands.command(
+        name="claim",
+        description="Claims an unclaimed tag. An unclaimed tag is a tag with no owner "
+        "because they have left the server.",
+    )
+    async def claim_tag(self, interaction: discord.Interaction, tag: str) -> None:
+        async with self.bot.database.transaction():
+            tag_record = await self.bot.database.fetchrow(
+                "SELECT * FROM Tags WHERE name = $1 AND guild_id = $2 AND deleted = FALSE;",
+                tag,
+                interaction.guild.id,
+            )
+
+            if tag_record:
+                try:
+                    member = await interaction.guild.fetch_member(
+                        tag_record["owner_id"]
+                    )
+                    if member is not None:
+                        await interaction.response.send_message(
+                            embed=discord.Embed(
+                                description=f"Owner of the tag **{tag}** is still present in the server.",
```

Comment:
Maybe include "the" before it? Also, to be consistent we should use quotes around the tag name. `f"The owner of the tag "{tag}" is still present in the server."`